Sources/Jetworking/Extensions/Data+Extension.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+extension Data {
+    init?(boundary: String, formData: [String: String], fileURL: URL, multipartFileContentType: MultipartContentType) {
```

Comment:
Please add a unit test for this function to validate that it is working correctly

Reply:
Done. Simple check is implemented within `DataExtensionTests`